packages/app-bridge-theme/src/types/Command.ts

```diff
@@ -1,31 +1,17 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
-import { type Simplify } from 'type-fest';
-
 import { type CommandRegistry } from '../registries';
 
-import { type ObjectNameValidator, type WrongNamePattern } from './Common';
-
-type CommandVerb = 'open' | 'close' | 'navigate' | 'download';
```

Comment:
You still want to have the verb as a gateway to prevent badly named commands (it is its only function)

Reply:
That still works. If I try to add command named `foo: string;` to `CommandRegistry` in brand-sdk, I get a warning that it does not match the pattern. And I can still add commands that match the pattern, e.g. `downloadSomtehing: string[];` Is there something I'm missing?

Reply:
Ahhh I didn't see you added them in the `CommandRegistry.ts`, all good from my side, sorry! 🙇